src/bp/core/services/module/resources-loader.ts

```diff
@@ -0,0 +1,107 @@
+import { Logger } from 'botpress/sdk'
+import fse from 'fs-extra'
+import path from 'path'
+
+import { addHashToFile, isOriginalFile } from '../../misc/checksum'
+import ModuleResolver from '../../modules/resolver'
+
+/** Describes a resource that the module wants to export to the bot */
+interface ModuleResourceExport {
+  [name: string]: FileTeleport
```

Comment:
can you document what the name is meant to mean or be used for?

Reply:
it was mostly meant to identify the type of resource to improve readability, but i'll check to refactor this part to make it simpler